tests/regression/82-widening_gas/04-side_simple_update.c

```diff
@@ -28,7 +34,7 @@ int main(void) {
   pthread_mutex_lock(&A);
   __goblint_check(a <= 1);
   __goblint_check(b <= 2);
-  __goblint_check(c <= 3);
+  __goblint_check(c <= 4); // UNKNOWN!
```

Comment:
The meaning of `//UNKNOWN!` in our regression suite is that the respective thing also does not hold in the concrete. I think the annotation to have here would be `//TODO`. Also, why 4?